config/checkstyle_checks.xml

```diff
@@ -159,6 +159,12 @@
                  value="All files in the ''src/it/java'' folder
                       should be named ''*Test.java'' or ''Abstract*.java''."/>
     </module>
+    <module name="RegexpSingleline">
+        <property name="id" value="oldFsfAddress"/>
+        <property name="format" value="\Q59 Temple Place\E"/>
+        <property name="message"
+                  value="The address of the FSF in licenses shall be ''51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA''."/>
```

Comment:
why we need this Check if we have update in `config/java_regexp.header`?
 any mismatch with header will be violation.

Reply:
The same address exists in many more files than just Java files (e.g. properties, html, xml, ...). Since I'm not sure where people copy such license texts from (other than the FSF website with the correct address) I'd like to avoid someone creating another outdated license.html in the future.

Reply:
It's not that people added the license as license header to these other files. It's just that other files contain pieces of the license text because licensing information is mandatory metadata e.g. for eclipse features or for the so called "branding plugin".